src/Microsoft.Azure.WebJobs.Host/ConfigurationUtility.cs

```diff
@@ -42,5 +42,20 @@ public static string GetConnectionFromConfigOrEnvironment(string connectionName)
 
             return Environment.GetEnvironmentVariable(connectionName) ?? configValue;
         }
+
+        public static bool IsSettingEnabled(string settingName)
```

Comment:
See Brett's PR https://github.com/Azure/azure-webjobs-sdk/pull/2513. This Type is going away